src/masternodeman.cpp

```diff
@@ -811,7 +811,7 @@ void CMasternodeMan::ProcessMessage(CNode* pfrom, std::string& strCommand, CData
                     int64_t t = (*i).second;
                     if (GetTime() < t) {
                         Misbehaving(pfrom->GetId(), 34);
-                        LogPrintf("dseg - peer already asked me for the list\n");
+                        LogPrint("masternode","dseg - peer already asked me for the list\n");
```

Comment:
post-merge: This should have been left as `LogPrintf()` as it results in a `Misbehaving` message.